test/fixtures/output/asyncapi/import-no-overwrite.golden

```diff
@@ -0,0 +1 @@
+"topic1" topic is already present and overwrite flag is false.
```

Comment:
No, this is not an error. We will skip that particular topic in case the overwrite flag is false. It can be used to only add new topics to the cluster and do not modify the existing ones. I am printing it as it is possible that there is no new topic in the input YAML. Without this print statement, there is nothing being printed in that case.